src/main/java/psi/model/CheckItem.java

```diff
@@ -0,0 +1,30 @@
+package psi.model;
+
+public class CheckItem implements Item {
+
+	private boolean value;
+	private ContainerItem items;
+
+	public CheckItem(boolean value, ContainerItem items) {
+		super();
+		this.value = value;
+		this.items = items;
+	}
+
+	public boolean isValue() {
```

Comment:
isChecked instead of isValue? :D
(naming comment, feel free to ignore)

Reply:
Fair point :+1: .